src/main/java/com/softserve/app/controller/ArticleController.java

```diff
@@ -0,0 +1,66 @@
+package com.softserve.app.controller;
+
+import com.softserve.app.constant.ArticleConstant;
+import com.softserve.app.dto.ArticleDTO;
+import com.softserve.app.models.Article;
+import com.softserve.app.service.ArticleService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
```

Comment:
Lombok Slf4j annotation and just log (log.info("")) instead of System.out.PrintLn()